gke/terraform.tfvars

```diff
@@ -1,8 +1,26 @@
 # SPDX-FileCopyrightText: Copyright (c) 2022-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-# Do not commit this file to Git if you add sensitive values
-# project_id = ""
-# cluster_name = ""
-# region     = "us-west1"
-# node_zones = ["us-west1-b"]
\ No newline at end of file
+# Sample tfvars file. Uncomment out values to use
+# Do not commit this file to Git with sensitive values
+
+# cluster_name                = ""
```

Comment:
To maintain consistency with other tfvars, can you add the cluster_name to be `holoscan-cluster`

Reply:
These are auto generated. The values which appear here are the defaults from `variables.tf`. The tfvars make it simple for people to select the values themselves